Moq.AutoMock/IAutoMockerDisposable.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Linq;
+
+namespace Moq.AutoMock
+{
+    /// <summary>
+    /// An interface that is used to clean up AutoMocker instances.
+    /// </summary>
+    public interface IAutoMockerDisposable : IDisposable
```

Comment:
What value is there in this being public?

Reply:
Same as above, allows for a "callback" to enable custom dispose logic.